src/main/java/org/apache/commons/compress/internal/lang3/SystemProperties.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.commons.compress.internal.lang3;
+
+public final class SystemProperties {
+
+  public static String getOsName() {
+    try {
+      return System.getProperty("os.name");
```

Comment:
A direct call to System.getProperty("os.name") would be simpler

Reply:
I wouldn't bother dealing with the SecurityManager, the `os.name` property is granted read access in the default policy anyway.

Reply:
Fixed in https://github.com/apache/commons-compress/pull/607/commits/18a4442972842ce8e456d49e0589acbe4bd14aef